{
  "name": "CVE-2019-18348",
  "seq": "2019-18348",
  "type": "CAN",
  "status": "Candidate",
  "phase": {
    "text": "Assigned",
    "date": "20191023"
  },
  "desc": "An issue was discovered in urllib2 in Python 2.x through 2.7.17 and urllib in Python 3.x through 3.8.0. CRLF injection is possible if the attacker controls a url parameter, as demonstrated by the first argument to urllib.request.urlopen with \\r\\n (specifically in the host component of a URL) followed by an HTTP header. This is similar to the CVE-2019-9740 query string issue and the CVE-2019-9947 path string issue. (This is not exploitable when glibc has CVE-2016-10739 fixed.). This is fixed in: v2.7.18, v2.7.18rc1; v3.5.10, v3.5.10rc1; v3.6.11, v3.6.11rc1, v3.6.12; v3.7.8, v3.7.8rc1, v3.7.9; v3.8.3, v3.8.3rc1, v3.8.4, v3.8.4rc1, v3.8.5, v3.8.6, v3.8.6rc1.",
  "refs": [
    {
      "text": "https://security.netapp.com/advisory/ntap-20191107-0004/",
      "source": "CONFIRM",
      "url": "https://security.netapp.com/advisory/ntap-20191107-0004/"
    },
    {
      "text": "FEDORA-2019-57462fa10d",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/4X3HW5JRZ7GCPSR7UHJOLD7AWLTQCDVR/"
    },
    {
      "text": "FEDORA-2019-b06ec6159b",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/M34WOYCDKTDE5KLUACE2YIEH7D37KHRX/"
    },
    {
      "text": "FEDORA-2019-d202cda4f8",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/JCPGLTTOBB3QEARDX4JOYURP6ELNNA2V/"
    },
    {
      "text": "FEDORA-2020-8bdd3fd7a4",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/UESGYI5XDAHJBATEZN3MHNDUBDH47AS6/"
    },
    {
      "text": "FEDORA-2020-ea5bdbcc90",
      "source": "FEDORA",
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/A5NSAX4SC3V64PGZUPH7PRDLSON34Q5A/"
    },
    {
      "text": "https://bugs.python.org/issue30458#msg347282",
      "source": "MISC",
      "url": "https://bugs.python.org/issue30458#msg347282"
    },
    {
      "text": "https://bugzilla.redhat.com/show_bug.cgi?id=1727276",
      "source": "MISC",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1727276"
    },
    {
      "text": "https://www.oracle.com/security-alerts/cpuoct2020.html",
      "source": "MISC",
      "url": "https://www.oracle.com/security-alerts/cpuoct2020.html"
    },
    {
      "text": "[debian-lts-announce] 20200715 [SECURITY] [DLA 2280-1] python3.5 security update",
      "source": "MLIST",
      "url": "https://lists.debian.org/debian-lts-announce/2020/07/msg00011.html"
    },
    {
      "text": "openSUSE-SU-2020:0696",
      "source": "SUSE",
      "url": "http://lists.opensuse.org/opensuse-security-announce/2020-05/msg00041.html"
    },
    {
      "text": "USN-4333-1",
      "source": "UBUNTU",
      "url": "https://usn.ubuntu.com/4333-1/"
    },
    {
      "text": "USN-4333-2",
      "source": "UBUNTU",
      "url": "https://usn.ubuntu.com/4333-2/"
    }
  ],
  "votes": {}
}
